### PR TITLE
Add Tuple/Dictionary episode, rearrange some other things very slightly

### LINKED ISSUE #4: Discuss dictionaries (and maybe tuples) with lists
Dictionaries are a powerful data structure - add to lists module

Note - probably missing right now because dataframes serve similar purpose

### PR DESCRIPTION
**Pull request by gonuke**
Mostly resolves #4 
Tuples/Dictionary episode could still benefit from some exercises, but this is ready for teaching...

### CODE REVIEW

File: _episodes/02-variables.md [hunk 1]
```diff
@@ -107,6 +123,45 @@ Age in three years: 45
 ~~~
 {: .output}
 
+## Avoid 'magic numbers'.
+
```
Comment (ChristinaLK) [inline _episodes/02-variables.md:127]: I love this example!  <3

**Review by ChristinaLK — APPROVED** [review]

**Comment by ChristinaLK** [comment]
Is the lunch episode actually happening at lunch?  Or the end of the day?

**Comment by gonuke** [comment]
Oh yeah... Lunch is our end of day - you're right...

**Comment by ChristinaLK** [comment]
can I merge this?

**Comment by gonuke** [comment]
Yes - go ahead.

Paul

-- ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ --
Paul P.H. Wilson
608-263-0807<tel:608-263-0807>
paul.wilson@wisc.edu<mailto:paul.wilson@wisc.edu>
419 Engineering Research Bldg<x-apple-data-detectors://0/3>
1500 Engineering Dr, Madison, WI 53706<x-apple-data-detectors://0/3>
calendar: http://go.wisc.edu/pphw-cal

Grainger Professor of Nuclear Engineering
Computational Nuclear Engineering Research Group

[Sent from my iPhone]

On Aug 29, 2017, at 20:46, Christina Koch <notifications@github.com<mailto:notifications@github.com>> wrote:


can I merge this?

**Pull request merged**